test/e2e/main/custom_configuration_test.go

```diff
@@ -200,3 +208,28 @@ xmlns:server="urn:infinispan:server:13.0">
 		Data: map[string]string{"infinispan-config." + format: userCacheContainer},
 	}
 }
+
+func addLog4jToCustomConfigMap(customConfigMap *corev1.ConfigMap) *corev1.ConfigMap {
```

Comment:
Last comment, I promise :slightly_smiling_face: 

The return type of this method is never used, so we can remove `*corev1.ConfigMap` from the signature and just do `customConfigMap.Data["log4j.xml"] = ...`